<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetodosIntervalo</class>
 <widget class="QMainWindow" name="MetodosIntervalo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metodos de intervalo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="interpolacion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>770</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolacion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grafico">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>770</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Gráfico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="biseccion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>680</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Bisección</string>
    </property>
   </widget>
   <widget class="QPushButton" name="puntoFijo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>680</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Punto fijo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Primera</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Primera</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Segunda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tercera</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cuarta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quinta</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Resultado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>941</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>531</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/ecuaciones4.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="primera1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="primera2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="primera3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="primera4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="primera5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="segunda1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="segunda2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="segunda3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="segunda4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tercera1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tercera2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tercera3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tercera4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tercera5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuarta1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuarta2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cuarta3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="quinta1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>610</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="quinta2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tolerancia">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="xl">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>420</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="xu">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tolerancia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>380</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Xl</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Xu</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
